core/server/data/migrations/versions/4.13/03-populate-members-products-events-table.js

```diff
@@ -0,0 +1,33 @@
+const {chunk} = require('lodash');
+const {createTransactionalMigration} = require('../../utils');
+const logging = require('@tryghost/logging');
+const ObjectId = require('bson-objectid').default;
+
+module.exports = createTransactionalMigration(
+    async function up(connection) {
+        logging.info('Adding members_product_events rows for existing members_products relationships');
+        const memberProductRelationships = await connection('members_products').select('*');
+
+        const memberProductEvents = memberProductRelationships.map((row) => {
+            return {
+                id: Reflect.construct(ObjectId, []).toHexString(),
```

Comment:
What's the reason for using `Reflect` here? 😛

Reply:
Ugh just me trying to get rid of errors in my editor! The types for ObjectId say that it needs to be called with new, so like `new ObjectId().toHexString()` - I wasn't actually supposed to push up the Reflect change :grin:  Was experimenting locally with other ways of calling it - will remove :grin:

Reply:
Have replaced!